Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,30 @@
 
 ## Exercise I
 
-a)
+a) O(n) Based on the size of the input data, it increases the running time
 
 
-b)
+b) O(n^2) For each value, it needs to compare to all other values in the list.
```

Comment:
We multiply the individual runtimes of each loop to obtain the runtime of an entire block since these loops are nested, giving us an overall runtime of O(n log(n)).
```
sum = 0
    for i in range(n):
      j = 1
      while j < n:
        j *= 2
        sum += 1
```